projects/mlops-for-agc.yml

```
@@ -0,0 +1,39 @@
+---
+name: Adopting running Analysis Grand Challenge (AGC) on Kubeflow MLOps platform
+postdate: 2025-02-18
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+experiments:
+  - ATLAS
+  - CMS
+  - HLLHC
+skillset:
+  - ML
+  - Python
```

```suggestion
  - Python
  - Kubernetes
```
Would this addition make sense?